Sources/SwiftDiagnostics/FixIt.swift

```diff
@@ -77,6 +77,14 @@ public struct FixIt: Sendable {
     case replaceTrailingTrivia(token: TokenSyntax, newTrivia: Trivia)
     /// Replace the child node of the given parent node at the given replacement range with the given new child node
     case replaceChild(data: any ReplacingChildData)
+    /// Replace the text within the given range in a source file with new text.
+    ///
+    /// Generally, one should use other cases to replace specific syntax nodes
+    /// or trivia, because it more easily leads to a correct result. However,
+    /// this case provides a fallback for textual replacement that ignores
+    /// syntactic structure. After applying a textual replacement, there is no
+    /// way to get back to a syntax tree without reparsing.
+    case textualReplacement(replacementRange: Range<AbsolutePosition>, sourceFile: SourceFileSyntax, newText: String)
```

Comment:
To match the other cases, this should start with `replace` as well. I would suggest `replaceText`.

```suggestion
    case replaceText(range: Range<AbsolutePosition>, with newText: String, in sourceFile: SourceFileSyntax)
```